pages/test_analytics/javascript_collectors.md

```diff
@@ -58,6 +58,7 @@ To configure Jest:
         "reporters": ["default", "buildkite-test-collector/jest/reporter"]
     }
     ```
+1. Ensure that you run Jest with the `--testLocationInResults` flag.
```

Comment:
Thanks for your contribution @malclocke 🎉 

Just a couple of suggestions ...

- Adding a space between before and after a code block, makes it more readable (as it stands out more) in the source.
- Since you're not instructing the reader to actually run Jest, but instead make a recommendation as part of running Jest, make this [a callout/note within a list](https://github.com/buildkite/docs/blob/main/styleguides/markdown-syntax-style.md#callouts-within-lists).

```suggestion

    **Note:** Ensure that you run Jest with the `--testLocationInResults` flag.
```

Reply:
Alternatively, you could make this part of step 1:

```
1. Make sure Jest runs with:

    * Access to [CI environment variables](/docs/test-analytics/ci-environments).
    * The `--testLocationInResults` flag.

1. Add `"buildkite-test-collector/jest/reporter"` to ...
```

Reply:
@gilesgas As I've been following this up I've realised this can be set as a configuration file option on the current version of Jest.  The project I was originally having problems with was using an older version which only supported this option as a CLI flag.
In the current version (v29.7.0) of Jest this works both as a config file option and a CLI flag.

So we could recommend the following in the configuration file instead, which would be a little cleaner:

``` json
    {
        "reporters": ["default", "buildkite-test-collector/jest/reporter"],
        "testLocationInResults": true,
    }
```

We already recommend setting this up this way in the [Javascript test collector README](https://github.com/buildkite/test-collector-javascript?tab=readme-ov-file#jest) so the instructions should probably be consistent here.

However, this is not mentioned in the [Jest docs for the configuration file](https://jestjs.io/docs/configuration) (it does work though) and I'm not sure how to word the instructions for this.  WDYT?